Applications/Spire/Include/Spire/KeyBindings/TagModel.hpp

```diff
@@ -0,0 +1,51 @@
+#ifndef SPIRE_TAG_MODEL_HPP
+#define SPIRE_TAG_MODEL_HPP
+#include <string>
+#include <vector>
+#include "Spire/KeyBindings/KeyBindings.hpp"
+#include "Spire/KeyBindings/Term.hpp"
+
+namespace Spire {
+
+	/*! \class TagModel
```

Comment:
Fix indentation.

Reply:
I will update the documentation in line with how classes are documented throughout Spire because I have noticed that they use one-liners instead of the "brief" tag.